metron-streaming/Metron-DataLoads/src/main/java/org/apache/metron/dataloads/hbase/mr/PrunerMapper.java

```diff
@@ -48,7 +48,7 @@ public void setup(Context context) throws IOException
         }
         try {
             tracker = AccessTrackerUtil.INSTANCE.loadAll(AccessTrackerUtil.INSTANCE.loadAll(table, atCF, atName, timestamp));
-        } catch (Exception e) {
+        } catch (Throwable e) {
```

Comment:
This is just a style thing.  It's going to throw a runtime exception
regardless and kill the task, but I wanted it to be clear in the logs that
it was due to not bing able to load the access tracker stuff (as opposed to
an OOM exception).

On Fri, Apr 1, 2016 at 1:18 PM, David Lyle <notifications@github.com> wrote:

> In
> metron-streaming/Metron-DataLoads/src/main/java/org/apache/metron/dataloads/hbase/mr/PrunerMapper.java
> <https://github.com/apache/incubator-metron/pull/66#discussion_r58236854>:
>
> > @@ -48,7 +48,7 @@ public void setup(Context context) throws IOException
> >          }
> >          try {
> >              tracker = AccessTrackerUtil.INSTANCE.loadAll(AccessTrackerUtil.INSTANCE.loadAll(table, atCF, atName, timestamp));
> > -        } catch (Exception e) {
> > +        } catch (Throwable e) {
>
> Was this changed to deliberately catch subclasses of Error?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-metron/pull/66/files/b4fc71e0a18ba9f2b897c3b01e16093903ed100a#r58236854>
>